### PR TITLE
Add readme for configuring swagger validation - REFAPP-281

### PR DESCRIPTION
**Pull request by robmckinnon**
Also add Kafka configuration steps, for logging validation results.

### CODE REVIEW

File: README-VALIDATION.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+To configure logging to Kafka, set the following env vars:
+
+```sh
+VALIDATION_KAFKA_TOPIC=kafka-test-topic
+VALIDATION_KAFKA_BROKER=127.0.0.1:9092
+```
+
+## Install local Kakfa install
+
```
Comment (ezodude) [inline README-VALIDATION.md]: Should we augment this to handle installing Kafka on other OSes?

Perhaps linking to an article with installation instructions :)

Comment (ezodude) [reply README-VALIDATION.md]: Just found this: https://kafka.apache.org/quickstart

**Comment by mbana** [comment]
https://github.com/bitnami/bitnami-docker-kafka
This part details how to create the topics: https://github.com/bitnami/bitnami-docker-kafka#step-3-create-the-third-node.

Just tried that out. Seems to work in docker.

versions available:

https://hub.docker.com/r/bitnami/kafka/tags/
https://hub.docker.com/r/bitnami/zookeeper/tags/

**Comment by ezodude** [comment]
@mbana Great find - thx!

**Pull request merged**